tier4_system_msgs/msg/DiagGraphStatus.msg

```diff
@@ -0,0 +1,4 @@
+builtin_interfaces/Time stamp
+tier4_system_msgs/DiagLeafStatus[] diags
+tier4_system_msgs/DiagUnitStatus[] units
+tier4_system_msgs/DiagLinkStatus[] links
```

Comment:
How are Status and Struct related?

Reply:
As shown in the figure "message representing the above graph":
- `diags[0:diag_size-1]` are status of `nodes[0:diag_size-1]`
- `units[0:unit_size-1]` are status of `nodes[diag_size:diag_size+unit_size-1]`

Reply:
Would it be better to simplify the structure by merging `diags` and `units`? It will consume an additional 12 bytes per node.

Reply:
Updated https://github.com/tier4/tier4_autoware_msgs/pull/113/commits/a23b1defedaa639578a86e5a0a2824b69dbe888e.